GeoBot/GeoBot/wwwroot/index.html

```diff
@@ -0,0 +1,87 @@
+<html>
+
+<body>
+    <div id="webchat" role="main"></div>
+    <script src="https://cdn.botframework.com/botframework-webchat/latest/webchat.js"></script>
+    <script>
+        function getUrlParam(name) {
+            var results = new RegExp('[\\?&]' + name + '=([^&#]*)').exec(window.location.href);
+            return (results && results[1]) || undefined;
+        }
+
+        var botname = getUrlParam('botname');
+        var endpoint = getUrlParam('endpoint');
+        
+        if (botname === undefined && endpoint === undefined) {
+            // if botname and endpoint is not set - we will use document.URL value as a default
+            hostname = document.location.hostname;
+            splitedHostname = hostname.split('.'); 
+            endpoint = "https://" + hostname;  
+            botname = splitedHostname[0];  
+        } 
+
+        function createFetchSpeechServicesCredentials(){
+            let expireAfter = 0;
+            let lastPromise;
+
+            return () => {
+                if (Date.now() > expireAfter) {
+                    const speechServiceUri = endpoint + "/speech/token";
+                    const speechServicesTokenResPromise = fetch(speechServiceUri, { method: 'GET' });
+
+                    expireAfter = Date.now() + 300000;
+                    lastPromise = speechServicesTokenResPromise.then(
+                        res => res.json(),
+                        err => {
+                            lastPromise = null;
+                            return Promise.reject(err);
+                        }
+                    );
+                }
+                return lastPromise;
+            }
+        }
+
+        async function getDirectlineToken(){
+            let tokenGenerateServiceUri = endpoint + "/directline/token";
```

Comment:
Same here, curious if this will work.

```suggestion
            let tokenGenerateServiceUri = "/directline/token";
```